src/views/ev3/l10n.json

```diff
@@ -34,5 +34,15 @@
     "ev3.otherComputerConnectedText": "Only one computer can be connected to an EV3 at a time. If you have another computer connected to your EV3, disconnect the EV3 or close Scratch on that computer and try again.",
     "ev3.updateFirmwareTitle": "Try updating your EV3 firmware",
     "ev3.updateFirmwareText": "We recommend updating to EV3 firmware version 1.10E or above. See {firmwareUpdateLink}.",
-    "ev3.firmwareUpdateText": "firmware update instructions from LEGO"
+    "ev3.firmwareUpdateText": "firmware update instructions from LEGO",
+    "ev3.imgAltEv3Illustration": "Illustration of an EV3 Computer featuring some examples of interacting with an EV3.",
+    "ev3.imgAltAddExtension": "In the editor, click on the \"Add Extensions\" button on the lower left.",
+    "ev3.imgAltAcceptConnection": "Use the buttons of your EV3 to accept the connection.",
+    "ev3.imgAltAcceptPasscode": "Use the center button of your EV3 to accept the passcode.",
+    "ev3.imgAltWaitForWindows": "Windows notifies you when EV3 is ready.",
+    "ev3.imgAltEnterPasscodeMac": "Enter the passcode into the connection request window opening on your Mac.",
+    "ev3.imgAltPlugInMotor": "Port A is not to be confused with Port 4 on the other side of the EV3.",
+    "ev3.imgAltWaveHello": "Example of a Scratch project with a waving fairy.",
```

Comment:
You can remove "example of" from all of these, so it's just "A Scratch project..."